src/qt/bitcoingui.cpp

```diff
@@ -360,6 +361,10 @@ void BitcoinGUI::createActions()
     m_create_wallet_action->setEnabled(false);
     m_create_wallet_action->setStatusTip(tr("Create a new wallet"));
 
+    m_restore_wallet_action = new QAction(tr("Restore Wallet…"), this);
+    m_restore_wallet_action->setEnabled(false);
+    m_restore_wallet_action->setStatusTip(tr("Restore a wallet from a backup file"));
```

Comment:
The strings being introduced need a translator comment. I have written documentation on how to construct these comments here: [qml/translator-comments.md](https://github.com/bitcoin-core/gui-qml/blob/main/src/qml/doc/translator-comments.md)

The difference here would be that instead of following the qml style, you'd be looking at how to mark a comment through the [qt widgets style](https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments).

For a multi-line comment, use `/*: ... */`. Let me know if you need help creating the translator comments.

Reply:
Done in a047c1a